test/cmd/newapp.sh

```diff
@@ -11,22 +11,21 @@ os::log::install_errexit
 
 # This test validates the new-app command
 
-os::cmd::expect_success 'oc create -f examples/image-streams/image-streams-centos7.json'
-
 os::cmd::expect_success_and_text 'oc new-app library/php mysql -o yaml' '3306'
 os::cmd::expect_failure 'oc new-app unknownhubimage -o yaml'
 # verify we can generate a Docker image based component "mongodb" directly
 os::cmd::expect_success_and_text 'oc new-app mongo -o yaml' 'library/mongo'
 # the local image repository takes precedence over the Docker Hub "mysql" image
+os::cmd::expect_success 'oc create -f examples/image-streams/image-streams-centos7.json'
 os::cmd::try_until_success 'oc get imagestreamtags mysql:latest'
 os::cmd::try_until_success 'oc get imagestreamtags mysql:5.5'
 os::cmd::try_until_success 'oc get imagestreamtags mysql:5.6'
-os::cmd::expect_success_and_text 'oc new-app mysql -o yaml' 'mysql'
+os::cmd::expect_success_and_not_text 'oc new-app mysql -o yaml' 'library/mysql'
```

Comment:
it's a not_text, i'm just making sure it did not pick up the library/mysql image, which it should not if the mysql imagestream is imported.